UserGuide.md

```diff
@@ -1,16 +1,28 @@
+
 # Auto Laser: Getting Started
 
 ## Prerequisites
-Auto Laser has several prerequisites you will have to satisfy before you are able to use the software.
+Auto Laser has a few prerequisites you will have to satisfy before you are able to use the software. 
+1. Download the PyCharm IDE and the Arduino IDE and drivers.
+2. Clone the "src" folder of this repository. 
+3. Open the autolaser.py file in PyCharm and build the project to make sure there are no build errors.
+4. Open the AutoLaser_ArduinoCode.ino file in the Arduino IDE and make sure there are no build errors.
+5. Connect the Arduino to your PC via the Serial port.
+6. Upload the AutoLaser_ArduinoCode.ino file to the arduino.
 
 ### Physical Assembly
-You must construct this 3d printed servo assembly which can be found [here](https://www.instructables.com/CheetahBeam-a-DIY-Automatic-Cat-Laser-Toy/). This is a necessary physical part which will control the laser assembly. Other supplies necessary will be a camera system to feed info to the system, and balloons to use as targets.
+You must construct this 3d printed servo assembly which can be found [here](https://www.instructables.com/CheetahBeam-a-DIY-Automatic-Cat-Laser-Toy/). This is a necessary physical part which will control the laser assembly. Other supplies necessary will be a camera to feed data into the system, and balloons to use as targets.
 
 ### Cloning the Repo
-You also must clone this repo using (git)[http://git-scm.com/], this will give you the software necessary to use and control the physical aspect of this project.
+You also must clone the "src" folder of this repo using [git](http://git-scm.com/), this will give you the software necessary to use and control the physical aspect of this project.
 
 ## Setting up the system
+Assemble the 3D-printed housing and the wiring for the servos and laser (follow the linked guide). Connect the arduino to your PC via the Serial port and set the port to COM3. Upload the AutoLaser_ArduinoCode.ino file to the arduino. Place the webcam directly underneath the laser. Now the system is ready to be run.
 
 ## Running the system
-
-## FAQ
+These are the steps for running the program:
+1. Open the autolaser.py code in PyCharm. 
```

Comment:
They don't need to open it in pycharm they can just run it from the command line using Python 3, They do need to pip install serial, cv2, pickle, and PySimpleGUI though, I'd just add those steps here and then alter the wording of 2 to specify run the code using Python 3